### PR TITLE
if build_image is used, it is also passed to worker build

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

**Comment by lcarva** [comment]
Looks great! Let's make sure this is unit tested.

**Comment by twaugh** [comment]
Are we happy that if neither build_image or build_imagestream is specified in the osbs.conf or in kwargs to osbs-client, that the plugin parameter will get set to `DEFAULT_BUILDROOT_IMAGE` (which is "buildroot:latest"? I wonder about this, because this will override any build_image set in the osbs.conf used by the orchestrator build to create the worker build.

**Comment by twaugh** [comment]
So actually the problem I mentioned above comes about only because we have a `DEFAULT_BUILDROOT_IMAGE`. Some time we'll remove that.

The code you have looks great once tests are in place.

File: tests/build_/test_build_request.py [hunk 1]
```diff
@@ -1044,8 +1044,17 @@ def test_render_optional_plugins(self, tmpdir):
         # Only one version defined so far
         DEFAULT_ARRANGEMENT_VERSION,
     ])
+    @pytest.mark.parametrize('build_image', [
```
Comment (lcarva) [inline tests/build_/test_build_request.py]: I think this should be more explicit on what the expected behavior is. 
This unit test is re-implementing logic that is being tested.
I think a stronger test would explicitly specify what the worker_build_image value should be depending on the values of `build_image` and `build_imagestream`.
Something along the lines of:

```python
@pytest.mark.parametrize(['build_image', 'build_imagestream', 'worker_build_image'], [
    ('fedora:latest', None, 'fedora:latest'),
    ...
])
```

**Comment by twaugh** [comment]
Any objections to merging? Until https://github.com/projectatomic/atomic-reactor/pull/669 is merged this parameter will be ignored.

**Comment by lcarva** [comment]
+1 for merging

**Comment by rcerven** [comment]
yop it can be safely merged

**Pull request merged**